tests/comfy_cli/command/nodes/test_node_install.py

```
@@ -145,6 +168,103 @@ def test_install_exit_on_fail_reraises_and_propagates_code():
+        mock_execute.side_effect = subprocess.CalledProcessError(-9, "cm-cli")
+        result = runner.invoke(app, ["install", "bad-node"])
+        assert result.exit_code != 0
+        assert isinstance(result.exception, subprocess.CalledProcessError)
+
+
+def test_install_exit_on_fail_json_stdout_carries_only_the_envelope(_reset_renderer):
+    """execute_cm_cli streams cm-cli's raw output to sys.stdout; in JSON mode
+    install routes that stream to stderr so stdout stays a single parseable
+    envelope, labeled with the actual subcommand."""
+
+    def _stream_then_fail(args, **kwargs):
```

Pylint (4.0.6)

[warning] Unused argument 'args'

(W0613)